assets/src/components/detours/diversionPanel.tsx

```diff
@@ -61,8 +61,8 @@ export const DiversionPanel = ({
           <h2 className="c-diversion-panel__h2">Detour Directions</h2>
           {directions ? (
             <ListGroup as="ol">
-              {directions.map((d) => (
-                <ListGroup.Item key={d.instruction} as="li">
+              {directions.map((d, i) => (
+                <ListGroup.Item key={i} as="li">
```

Comment:
`i` in this context is the index, right?

If so, then I _think_ React recommends against using the index as a key. I'm not an expert here, so I could easily see this exact use case not really being a problem, but it's still something that I've sort of been trained to shy away from.

https://react.dev/learn/rendering-lists#why-does-react-need-keys